docs/partials/apiExamples.md

```diff
@@ -197,6 +197,26 @@ The following example illustrates how a user may query the `element.value` or `e
 
 </auro-accordion>
 
+### Custom Validity with Error State
```

Comment:
There is something that broke with recent updates. It would be nice if we could get that addressed too since we are adding to this API demo page. 

https://github.com/AlaskaAirlines/auro-select/issues/146